index.html

```diff
@@ -10,189 +11,223 @@
+                <li class="lang-container">
+                    <img class="lang-img" src="req/html.png" alt="html">
+                    <p class="lang">HTML</p>
                 </li>
-                <li>
-                    <a href="#"><img src="req/V.png" alt="V"></a>
+                <li class="lang-container">
+                    <img class="lang-img" src="req/css.png" alt="css">
+                    <p class="lang">CSS</p>
                 </li>
             </ul>
-        </div>
-        <a href="#" class="link">Get my resume</a>
-    </div>
-    <ul class="about-skills">
-    <div>
-        <li class="lang-detail">
-        <h3 class="lang-head">Languages</h3>
-        <i class="arrow down"></i> 
-        </div>
-        <ul class="lang-names">
-            <li class="lang-container">
-                <img class="lang-img" src="req/js.png" alt="javascript">
-                <p class="lang">javascript</p>
             </li>
-            <li class="lang-container">
-                <img class="lang-img" src="req/html.png" alt="html">
-                <p class="lang">HTML</p>
+            <li class="lang-detail bottom-line">
+                <h3 class="Lang-head">Frameworks</h3>
+                <i class="arrow right"></i>
             </li>
-            <li class="lang-container">
-                <img class="lang-img" src="req/css.png" alt="css">
-                <p class="lang">CSS</p>
+            <li class="lang-detail bottom-line">
+                <h3 class="Lang-head">Skills</h3>
+                <i class="arrow right"></i>
             </li>
         </ul>
-    </li>
-        <li class="lang-detail bottom-line">
-            <h3 class="Lang-head">Frameworks</h3>
-            <i class="arrow right"></i> 
-        </li>
-        <li class="lang-detail bottom-line">
-            <h3 class="Lang-head">Skills</h3>
-            <i class="arrow right"></i>    
-        </li>
-    </ul>
-</section>
+    </section>
+
+    <section class="container">
+        <div class="contact-info">
+          <h2 class="container-header">contact me</h2>
+          <p class="container-text">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+
+        <form method="POST" action="https://formspree.io/f/xpzedjlw" class="contact-form">
+          <input type="text" name="name" placeholder="Muneeb"  maxlength="30" required>
+          <input
+            type="email"
+            name="email"
+            placeholder="example@gmail.com" required
+          >
+          <textarea
+            name="message" class="center-placeholder" maxlength="500" placeholder="Write your message here" required
+          ></textarea>
+          <button class="btn" type="submit" >Get in Touch</button>
+        </form>
+      </section>
 
```

Comment:
- [ ] You did a great job so far on this project , but you forget to add the inline bar below your get in touch bouton , kindly add for your project to match the figma design and be more attractive :wink:  try to fix the font on your design and make it as we can see on the figma design :smile: 

Your Design :point_down: 

![Screenshot from 2023-02-22 00-31-45](https://user-images.githubusercontent.com/88894525/220474422-066794f3-4bca-4f6c-a221-8c7082448453.png)

Figma Design :point_down: 
![figmalinebar](https://user-images.githubusercontent.com/88894525/220474524-d7588c99-e0eb-4b27-a2fe-e524118d5d0b.png)